tools/nnyeah/nnyeah/Program.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using System.IO;
+
+namespace nnyeah {
+    class Program {
+        static void Main (string [] args)
+        {
+            if (args.Length != 2) {
+                Console.Error.WriteLine ("Usage: mono nnyeah.exe /path/to/input/file.dll /path/to/output/file.dll");
+                Environment.Exit (1);
+            }
+            using (var stm = new FileStream (args [0], FileMode.Open, FileAccess.Read, FileShare.ReadWrite)) {
```

Comment:
Not a blocker, you could have done:
```suggestion
            using var stm = new FileStream (args [0], FileMode.Open, FileAccess.Read, FileShare.ReadWrite)
```

And remove the need of the indentation.